lib/servers/abstractservers/script_scanner/experiment.py

```diff
@@ -21,17 +21,20 @@ def _connect(self):
             try:
                 self.cxn = labrad.connect()
             except Exception as e:
-                raise Exception ("Not able to connecto LabRAD")
+                raise Exception("Not able to connect to LabRAD")
+        try:
+            self.sc = self.cxn.servers['ScriptScanner']
+        except Exception as e:
```

Comment:
For more details, see:

https://docs.python.org/2/tutorial/errors.html#handling-exceptions